src/main/java/edu/kis/vh/nursery/structures/IntArrayStack.java

```diff
@@ -26,31 +25,31 @@ public void push(final int IN) {
      * @return return true, if stack is empty and false, if it's not.
      */
     public boolean isEmpty() {
-        return total == MIN;
+        return total <= EMPTY_STRUCTURE_INDEX;
```

Comment:
Changing EMPTY_STRUCTURE_INDEX value requires such fixes? It does not make sense. Incorrect fix.
ERROR: total variable initialization and validation should always remain -1.